node/auth/authenticator.go

```diff
@@ -139,7 +140,7 @@ func (a *requestAuthenticator) getDisperserKey(
 	disperserID uint32) (*gethcommon.Address, error) {
 
 	if !a.disperserIDFilter(disperserID) {
-		return nil, fmt.Errorf("invalid disperser ID: %d", disperserID)
+		return nil, api.NewErrorInvalidArg(fmt.Sprintf("invalid disperser ID: %d", disperserID))
```

Comment:
there's no mention of GRPC in this method name, so I had to go check whether its wrapping a grpc error code or not. Should we change the name to api.NewGRPCErrorInvalidArg or something now that we have many different error handling types/functions in api.?

Reply:
The errors in `api` are intended for APIs, which is gRPC. Unless there are non gRPC APIs I think it should be clear.